src/feature/client/dnsserv.c

```diff
@@ -213,6 +214,9 @@ dnsserv_launch_request(const char *name, int reverse,
   edge_connection_t *conn;
   char *q_name;
 
+  /* Launching a request for a user counts as user activity. */
+  note_user_activity(approx_time());
```

Comment:
I think that the other ones maybe shouldn't. I think just a DNS request (on the dnsport) should trigger this.  Maybe we need more discussion.

Reply:
For instance, once we'll have `HSFETCH` for v3, without a live consensus, it will most likely fail if tor is dormant and we let it dormant on that comment.

The other one that will is `ADD_ONION`.